### ISSUE TITLE
Failed to create directory property 'targetDir'

### ISSUE THREAD

**Issue by lukaspj** [JetBrains/gradle-grammar-kit-plugin#61]
Hi,

I'm trying to upgrade to the latest version of this plugin but I get this error:
```
Execution failed for task ':generateLexer'.
> Failed to create directory 'C:\Users\lukas\Development\Torque3D\IDEA\task ':generateLexer' property 'targetDir''
```

This looks to me like somewhere is missing a call to .get.

I have attached my build.gradle.kts file.

I tried looking through the code in master, but couldn't find any obvious mistake, I'm probably missing something.

[build.gradle.kts.txt](https://github.com/JetBrains/gradle-grammar-kit-plugin/files/7699035/build.gradle.kts.txt)

**Comment by YannCebron** [comment]
what is this line in `outputs.file("${targetDir}/${targetClass}.java")` ?
why do you have both `val generateTorqueScriptLexer =...` as well as `generateLexer` task?

see this as reference https://github.com/YannCebron/m68kplugin/blob/main/build.gradle.kts

**Comment by lukaspj** [comment]
Right sorry, I had copy-pasted my "try something and see if it gives me another error" version of the build.gradle.kts.

So, I got it running. I had to stop using the `val generateTorqueScriptLexer =...` version and instead use the `generateLexer` version. Otherwise it would keep trying to run the unconfigured `generateLexer` task instead of my version of the task. (The named version of the generate tasks was inspired by [intellij-rust](https://github.com/intellij-rust/intellij-rust/blob/master/build.gradle.kts#L388))

Then I had to change the outputs to
`outputs.file("${targetDir.get()}/${targetClass.get()}.java")` since apparently something is different there I guess.

Thanks a ton! :)
